server/monitor.go

```diff
@@ -2211,3 +2211,264 @@ func (s *Server) accountInfo(accName string) (*AccountInfo, error) {
 		responses,
 	}, nil
 }
+
+// LeafzOptions are options passed to Jsz
+type JSzOptions struct {
+	Account  string `json:"account,omitempty"`
+	Accounts bool   `json:"accounts,omitempty"`
+	Streams  bool   `json:"streams,omitempty"`
+	Consumer bool   `json:"consumer,omitempty"`
+	Offset   int    `json:"offset,omitempty"`
+	Limit    int    `json:"limit,omitempty"`
+}
+
+type StreamDetail struct {
+	Name     string          `json:"name"`
+	Cluster  *ClusterInfo    `json:"cluster,omitempty"`
+	Config   StreamConfig    `json:"config,omitempty"`
+	State    StreamState     `json:"state,omitempty"`
+	Consumer []*ConsumerInfo `json:"consumer_detail,omitempty"`
+}
+
+type AccountDetail struct {
+	Name string `json:"name"`
+	Id   string `json:"id"`
+	JetStreamStats
+	Streams []StreamDetail `json:"stream_detail,omitempty"`
+}
+
+// LeafInfo has detailed information on each remote leafnode connection.
+type JSInfo struct {
+	ID       string          `json:"server_id"`
+	Now      time.Time       `json:"now"`
+	Disabled bool            `json:"disabled,omitempty"`
+	Config   JetStreamConfig `json:"config,omitempty"`
+	JetStreamStats
+	StreamCnt    int          `json:"total_streams,omitempty"`
+	ConsumerCnt  int          `json:"total_consumers,omitempty"`
+	MessageCnt   uint64       `json:"total_messages,omitempty"`
+	MessageBytes uint64       `json:"total_message_bytes,omitempty"`
+	Meta         *ClusterInfo `json:"meta_cluster,omitempty"`
+	// aggregate raft info
+	StreamsNonReplicatedCnt   uint64           `json:"total_streams_non_replicated,omitempty"`
+	StreamsInQuorumCnt        uint64           `json:"total_streams_in_quorum,omitempty"`
+	StreamsOutOfQuorumCnt     uint64           `json:"total_streams_out_quorum,omitempty"`
+	ConsumersNonReplicatedCnt uint64           `json:"total_consumers_non_replicated,omitempty"`
+	ConsumersInQuorumCnt      uint64           `json:"total_consumers_in_quorum,omitempty"`
+	ConsumersOutOfQuorumCnt   uint64           `json:"total_consumers_out_quorum,omitempty"`
```

Comment:
we need to think a bit about the in/out terminology here, not sure the distintion really matters to be honest, lets add a total count for now and later if we need we can add a count for specifics

Reply:
removed them for streams and consumer. stream/consumer count was already present